docker/README.md

```diff
@@ -10,8 +10,11 @@ We build, assemble, and test our artifacts on docker containers. All of our pipe
 
 If you only want to build the docker image for either x64 or arm64, run this on a x64 or arm64 host respectively:
 
+You need to run the script within the `opensearch-build/docker/ci` folder. Running it
```

Comment:
Don't fold at 80 chars? Maybe merge with the sentence above:

"To build the docker image for either x64 or arm64, run `docker` on a x64 or arm64 host respectively within the `pensearch-build/docker/ci` folder:"

Reply:
> Don't fold at 80 chars? Maybe merge with the sentence above:
> 
> "To build the docker image for either x64 or arm64, run `docker` on a x64 or arm64 host respectively within the `pensearch-build/docker/ci` folder:"

Yeap sure, edited instruction.